### ISSUE TITLE
Can't get undercurl to work

### ISSUE THREAD

**Issue by npearson72** [alacritty/alacritty#6283]
Here's my undercurl test in zsh:

<img width="371" alt="Screen Shot 2022-08-24 at 4 35 29 PM" src="https://user-images.githubusercontent.com/1323414/186541713-2bfaac65-5374-4155-9923-25e51e19a27c.png">

### System

OS: macOS
Version: 0.10.1 (2844606)

### Logs

Crashes: STDERR, STDOUT
Font/Terminal size: `alacritty -vv`
Keyboard and bindings: `alacritty --print-events`

Logs attached:

[Alacritty-88457.log](https://github.com/alacritty/alacritty/files/9420035/Alacritty-88457.log)
[Alacritty-88414.log](https://github.com/alacritty/alacritty/files/9420036/Alacritty-88414.log)

Here are my alacritty.yml configs:

```
env:
  TERM: alacritty

window:
  startup_mode: SimpleFullscreen
  decorations: none

font:
  size: 14
  normal:
    family: Hack Nerd Font Mono
    style: Regular
  bold:
    family: Hack Nerd Font Mono
    style: Bold
  italic:
    family: Hack Nerd Font Mono
    style: Italic
  offset:
    x: 0
    y: 3

  use_thin_strokes: true

colors:
 # Default Colors
 primary:
   background: '#212121'
   foreground: '#f1f1f1'
 # Normal colors
 normal:
   black:   '#212121'
   red:     '#c30771'
   green:   '#10a778'
   yellow:  '#a89c14'
   blue:    '#008ec4'
   magenta: '#523c79'
   cyan:    '#20a5ba'
   white:   '#f1f1f1'
 # Bright colors
 bright:
   black:   '#424242'
   red:     '#e32791'
   green:   '#5fd7a7'
   yellow:  '#f3e430'
   blue:    '#b6d6fd'
   magenta: '#6855de'
   cyan:    '#4fb8cc'
   white:   '#ffffff'

 cursor:
   cursor: '#1ac7fc'

# background_opacity: 0.99

bell:
  duration: 10
  color: '#626262'

# Use `xxd -psd` to discover other chars hex values
key_bindings:
  - { key: H,         mods: Alt,        chars: "\x1b\x68" }
  - { key: J,         mods: Alt,        chars: "\x1b\x6a" }
  - { key: K,         mods: Alt,        chars: "\x1b\x6b" }
  - { key: K,         mods: Command,    chars: "\x1C"     } # See .tmux.conf
  - { key: L,         mods: Alt,        chars: "\x1b\x6c" }
  - { key: O,         mods: Alt,        chars: "\x1b\x6f" }
  - { key: R,         mods: Shift|Alt,  chars: "\x1b\x52" }
  - { key: R,         mods: Alt,        chars: "\x1b\x72" }
  - { key: S,         mods: Alt,        chars: "\x1b\x73" }
  - { key: T,         mods: Alt,        chars: "\x1b\x74" }
  - { key: W,         mods: Alt,        chars: "\x1b\x77" }
  - { key: Backslash, mods: Shift|Alt,  chars: "\x1b\x7c" }
  - { key: Minus,     mods: Shift|Alt,  chars: "\x1b\x5f" }
  - { key: LBracket,  mods: Alt,        chars: "\x1b\x5b" }
  - { key: RBracket,  mods: Alt,        chars: "\x1b\x5d" }
  - { key: Semicolon, mods: Shift|Alt,  chars: "\x1b\x3a" }
  - { key: Slash,     mods: Alt,        chars: "\x1b\x2f" }

  - {
      key: N,
      mods: Command,
      command: {
        program: "open",
        args: ["-n", "/Applications/Alacritty.app"]
      }
    }

  - {
      key: Return,
      mods: Command,
      action: ToggleSimpleFullscreen
    }
```

Here are my tmux configs

```
set -g default-terminal "alacritty"
set -as terminal-overrides ',*:Smulx=\E[4::%p1%dm' # undercurl support
set -as terminal-overrides ',*:Setulc=\E[58::2::%p1%{65536}%/%d::%p1%{256}%/%{255}%&%d::%p1%{255}%&%d%;m' # underscore colours - needs tmux-3.0
```

**Comment by chrisduerr** [comment]
See changelog, it will be released in 0.11.0.

**Comment by npearson72** [comment]
My bad. Thank you!
